raiden/services.py

```diff
@@ -83,11 +83,11 @@ def update_monitoring_service_from_balance_proof(
         sender=channel_state.our_state, receiver=channel_state.partner_state
     )
 
-    if balance < MONITORING_MIN_CAPACITY:
+    if balance < MONITORING_REWARD:
```

Comment:
This is the wrong balance, i.e. the balance of the channel that has nothing to do with the UDC balance.